tests/pivots/spreadsheet_pivot/spreadsheet_pivot.test.ts

```diff
@@ -446,6 +446,14 @@ describe("Spreadsheet Pivot", () => {
     expect(getEvaluatedGrid(model, "B26:F26")).toEqual([["5", "9", "14", "Total", ""]]);
   });
 
+  test("Cannot use PIVOT function inside its range", () => {
```

Comment:
Not sure what will be merged first, but we should do the same thing for `PIVOT.VALUE/PIVOT.HEADER` formulas (and have tests). Idk if @LucasLefevre handled that correctly in his PR ?